src/adapt/layout.js

```diff
@@ -1526,7 +1607,17 @@ adapt.layout.Column.prototype.layoutBreakableBlock = function(nodeContext) {
         // TODO: should this be done after first-line calculation?
         var edge = self.calculateEdge(resNodeContext, checkPoints, checkPointIndex,
             checkPoints[checkPointIndex].boxOffset);
-        var overflown = self.isOverflown(edge);
+        var overflown = false;
+        if (!adapt.layout.isOrphan(resNodeContext.viewNode)) {
```

Comment:
Fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/323/commits/f9b3bd77aa56e71586c8bee8911318f62b6b9685)